archive/archive/index.html

```diff
@@ -4,7 +4,7 @@
     <a tabindex=-1 role=menuitem id=dark-theme-item>Dark Theme</a><div role=separator></div><a tabindex=-1 role=menuitem id=syntax-coloring-item>Color Examples</a><div role=separator></div><a tabindex=-1 role=menuitem href=https://github.com/istio/istio.io/edit/release-1.6/content/en/_index.html>Edit this Page on GitHub</a>
     <a tabindex=-1 role=menuitem href="https://github.com/istio/istio.io/issues/new?title=Issue%20with%20_index.html">Report Site Bugs</a><div role=separator></div><h6>Other versions of this site</h6><a tabindex=-1 role=menuitem onclick="navigateToUrlOrRoot('https://istio.io/\/');return false;">Current Release</a>
     <a tabindex=-1 role=menuitem onclick="navigateToUrlOrRoot('https://preliminary.istio.io/\/');return false;">Next Release</a>
-    <a tabindex=-1 role=menuitem href=https://archive.istio.io>Older Releases</a></div></div><button id=search-show title="Search this site" aria-label=Search><svg class="icon"><use xlink:href="./img/icons.svg#magnifier"/></svg></button></div><form id=search-form name=cse role=search><input type=hidden name=cx value=002184991200833970123:iwwf17ikgf4>
+    <a tabindex=-1 role=menuitem href=https://istio.io/archive>Older Releases</a></div></div><button id=search-show title="Search this site" aria-label=Search><svg class="icon"><use xlink:href="./img/icons.svg#magnifier"/></svg></button></div><form id=search-form name=cse role=search><input type=hidden name=cx value=002184991200833970123:iwwf17ikgf4>
```

Comment:
I think this is also broken on all the`archive/vX.Y/index.html` pages too.

Reply:
Looks like >= V1.4 work correctly while <= V1.3 fail. It's going to take a while to go back and rebuild all of those site, so it may be easier to add a redirect from archive.istio.io to istio.io/archive

Reply:
Hmm, would it be fixed if you did a grep for `archive.istio.io` to find all the index.html files that are broken and make this same change to all of them? Or is there more to it than that?